hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -355,6 +355,8 @@ object DataSourceWriteOptions {
   val HIVE_TABLE_PROPERTIES = "hoodie.datasource.hive_sync.table_properties"
   val HIVE_TABLE_SERDE_PROPERTIES = "hoodie.datasource.hive_sync.serde_properties"
   val HIVE_SYNC_AS_DATA_SOURCE_TABLE = "hoodie.datasource.hive_sync.sync_as_datasource"
+  val HIVE_CREATE_MANAGED_TABLE = "hoodie.datasource.hive_sync_create_managed_table"
```

Comment:
`hoodie.datasource.hive_sync.create_managed_table` is better?

Reply:
done!